packages/html/src/tooltip/tests/tooltip-callout-position.tsx

```diff
@@ -0,0 +1,158 @@
+            <span>Callout right</span>
+            <span>Callout left</span>
+
+            <span>Content</span>
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal callout="bottom" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal callout="top"/>
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal callout="right"/>
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal callout="left"/>
+                </AnimationContainer>
+            </section>
+
+            <span>Long content</span>
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" callout="bottom" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" callout="top" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" callout="right" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" callout="left" />
+                </AnimationContainer>
+            </section>
+
+            <span>Content & Title</span>
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal title="Tooltip title" callout="bottom" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal title="Tooltip title" callout="top" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal title="Tooltip title" callout="right" />
+                </AnimationContainer>
+            </section>
+
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal title="Tooltip title" callout="left" />
+                </AnimationContainer>
+            </section>
+
+            <span>Long content & Title (with glyphs)</span>
+            <section>
+                <AnimationContainer>
+                    <TooltipNormal content="long tooltip content. Enough so it goes on another line" title="g y p q f t d l" callout="bottom" />
```

Comment:
This is purely a cosmetic suggestion.
```suggestion
                    <TooltipNormal content="Long tooltip content. Enough so it goes on another line" title="g y p q f t d l" callout="bottom" />
```
Some long content sentences begin with uppercase L, others with lowercase L.
(appears in **tooltip.tsx** and **tooltip-rtl.tsx** as well)